src/Optimization.js

```diff
@@ -24,4 +25,38 @@ module.exports = class extends ChainedMap {
       'usedExports',
     ]);
   }
+
+  minimizer(name) {
+    if (!this.minimizers.has(name)) {
```

Comment:
I think this could be changed to use `getOrCompute()`, like `Config.js` uses for `.plugin()`:
https://github.com/neutrinojs/webpack-chain/blob/3fe26f00c6b0ea7c8a481470934f719e8344e564/src/Config.js#L113-L115